content/blog/architecture-as-code-intro/index.md

```diff
@@ -0,0 +1,307 @@
+
+A key part of serverless architecture is an API Gateway that provides logical routes for mapping standard HTTP operations such as GET, PUT, POST, and DELETE to functions. The gateway makes development easier since these are standard and well-known interfaces. In addition to an API Gateway, the serverless platform should provide REST endpoints that allow you to manage the deploy with a CLI, portal, or an automation script. Finally, serverless architecture is extensible and supports integration with event sources and resources from the cloud provider through webhooks and other mechanisms.
+
+The following example illustrates how to deploy two functions, one written in Python and the other written in Go. You can find the [full example](https://github.com/pulumi/examples/tree/master/gcp-ts-serverless-raw) in the Pulumi Github repository.
+
+```ts
+import * as gcp from "@pulumi/gcp";
+import { asset } from "@pulumi/pulumi";
+
+const bucket = new gcp.storage.Bucket("bucket");
+
+// Google Cloud Function in Python
+
+const bucketObjectPython = new gcp.storage.BucketObject("python-zip", {
+    bucket: bucket.name,
+    source: new asset.AssetArchive({
+        ".": new asset.FileArchive("./pythonfunc"),
+    }),
+});
+
+const functionPython = new gcp.cloudfunctions.Function("python-func", {
+    sourceArchiveBucket: bucket.name,
+    runtime: "python37",
+    sourceArchiveObject: bucketObjectPython.name,
+    entryPoint: "handler",
+    triggerHttp: true,
+    availableMemoryMb: 128,
+});
+
+export const pythonEndpoint = functionPython.httpsTriggerUrl;
+
+// Google Cloud Function in Go
+
+const bucketObjectGo = new gcp.storage.BucketObject("go-zip", {
+    bucket: bucket.name,
+    source: new asset.AssetArchive({
+        ".": new asset.FileArchive("./gofunc"),
+    }),
+});
+
+const functionGo = new gcp.cloudfunctions.Function("go-func", {
+    sourceArchiveBucket: bucket.name,
+    runtime: "go111",
+    sourceArchiveObject: bucketObjectGo.name,
+    entryPoint: "Handler",
+    triggerHttp: true,
+    availableMemoryMb: 128,
+});
+
+export const goEndpoint = functionGo.httpsTriggerUrl;
+```
+
+## Kubernetes
+
+Kubernetes is a container orchestration system for deploying, scaling, and managing your application. Applications use multiple containers that contain code and dependencies needed for the code to execute. Kubernetes runs on a cluster of nodes and containers are deployed in pods. Kubernetes manages the pods by replacing containers that fail and killing containers that don't respond. It can also scale the number of containers as needed and load balance the network traffic. These are some of the features that Kubernetes provides:
```

Comment:
Is "killing" the right word? If it's not a Kubernetes-specific term, maybe something less aggressive, like destroying or disposing of.

Reply:
substituted with remove